src/ucp/core/ucp_request.inl

```diff
@@ -114,6 +114,15 @@ ucp_request_complete_stream_recv(ucp_request_t *req,
     ucp_request_complete(req, recv.stream.cb, status, req->recv.stream.length);
 }
 
+static UCS_F_ALWAYS_INLINE int
```

Comment:
minor: i'd add `recv` to the name of this func.  
Maybe `ucp_request_can_complete_stream_recv`, because you already have `ucp_request_complete_stream_recv`